index.html

```diff
@@ -56,9 +55,293 @@ <h2 class="connect">LET'S CONNECT</h2>
+                                        src="./images/Separator-Bottom.png"
+                                        alt=""
+                                        class="separator"
+                                      />
+                                      <div class="event18">
+                                        <h3>Skills</h3>
+                                        <button type="submit" class="dropbtn">
+                                          <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                                        </button>
+                                      </div>
+                                      <img
+                                        src="./images/Separator-Bottom.png"
+                                        alt=""
+                                        class="separator"
+                                      />
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                              <div class="contact-edith">
+                                <h2>Contact Me</h2>
+                                <p>
+                                  If you have an application you are interested in developing, a feature
+                                  that you need built or a project that needs coding. I’d love to help
+                                  with it
+                                </p>
+                                <form action="https://formspree.io/f/mqknvnyz" method="post">
+                                  <input
+                                    type="text"
+                                    id="username"
+                                    required
+                                    maxlength="30"
+                                    placeholder="Text input"
+                                    name="userName"
+                                  />
+                                  <input
+                                    type="email"
+                                    id="email"
+                                    required
+                                    placeholder="obiba@gmail.com"
+                                    name="userEmail"
+                                  />
+                                  <textarea
+                                    name="details"
+                                    id="detail"
+                                    placeholder="Write your message here"
+                                    required
+                                    maxlength="500"
+                                  ></textarea>
+                                  <button class="form-button go-newmes" type="submit">
+                                    Get in touch
+                                  </button>
+                                </form>
+                                ​
```

Comment:
- [ ] Kindly remove this part of the document for now. You can start it somewhere till you need it. There will be a separate project milestone for them.